src/libs/Navigation/PlatformStackNavigation/navigationOptions/animation/none/index.native.ts

```diff
@@ -0,0 +1,6 @@
+import type {NativeStackNavigationOptions} from '@react-navigation/native-stack';
+import Animations from '..';
+
+const none: NativeStackNavigationOptions = {animation: Animations.NONE, gestureEnabled: false};
+
+export default none;
```

Comment:
@fabioh8010 `PlatformSpecificNavigationOptions` is a generic type. I don't see a lot of sense to create a typealias for this type. Could you elaborate which advantages it can give and why we can not use generic-purpose-navigator type here? 🤔

Reply:
Hi @kirillzyusko, the reason for this is for file/type organization purposes, as defined in our [TS Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#file-organization). By creating a shared `types.ts` for utility funtcions that have platform variants like this one, we ensure the function typing between all files are consistent. It's a common pattern we have for other utility / lib files too.

Reply:
@fabioh8010 ah, I see what you meant! The only thing is that we use different types for different platforms I think kind of intentionally. If we use `PlatformSpecificNavigationOptions` (which act as a union between `StackNavigationOptions` and `NativeStackNavigationOptions`) for both files, then we can accidentally include `stack` navigator-props for `native-stack` which will be ignored and will just increase bundle size.

When we use different types between different files we are kind of assuring that we will not use web-props in native files. Does it makes sense what I'm saying?